### PR TITLE
Replace access token with api keys

### PR DESCRIPTION
**Pull request by ankit01-oss**

### CODE REVIEW

File: data/docs/alerts-management/terraform-provider-signoz.mdx [hunk 1]
```diff
@@ -25,7 +25,7 @@ This document will show you how to:
   - version of signoz provider: [0.0.4](https://github.com/SigNoz/terraform-provider-signoz/releases/tag/v0.0.4) or later
 - SigNoz Cloud
   - endpoint URL
-  - access token
```
Comment (therealpandey) [inline data/docs/alerts-management/terraform-provider-signoz.mdx:28]: This needs to be api-key

File: data/docs/alerts-management/terraform-provider-signoz.mdx [hunk 2]
```diff
@@ -65,15 +65,15 @@ provider "signoz" {}
 
 By setting the environmental variables below, the signoz provider will automatically read these variables and authorize with your SigNoz Cloud account:
 ``` bash
-SIGNOZ_INGESTION_KEY="your-access-token"
+SIGNOZ_INGESTION_KEY="your-ingestion-key"
 SIGNOZ_ENDPOINT="https://your-signoz-endpoint"
 ```
 
-You can explicitly set these variables in the `provider {}` block as shown below, but avoid hardcoding your access token if you plan to commit your `.tf` files to an online version control system.
+You can explicitly set these variables in the `provider {}` block as shown below, but avoid hardcoding your ingestion key if you plan to commit your `.tf` files to an online version control system.
```
Comment (therealpandey) [inline data/docs/alerts-management/terraform-provider-signoz.mdx]: Api Key

File: data/docs/alerts-management/terraform-provider-signoz.mdx [hunk 3]
```diff
@@ -65,15 +65,15 @@ provider "signoz" {}
...
 ```hcl
 provider "signoz" {
   endpoint="https://your-signoz-endpoint"
-  access_token="your-access-token"  # Do not type your token directly if you push codes online
+  INGESTION_KEY="your-ingestion-key"  # Do not type your token directly if you push codes online
```
Comment (therealpandey) [inline data/docs/alerts-management/terraform-provider-signoz.mdx]: Api key

It is actually access_token = <api_key>

File: data/guides/how-to-authenticate-and-embedded-grafana-charts-into-iframe.mdx [hunk 4]
```diff
@@ -194,7 +194,7 @@ Grafana supports several secure authentication methods for embedded dashboards.
                     <title>Embed Grafana Chart</title>
                 </head>
                 <body>
-                    <iframe src="https://grafana.example.com/d/dashboard-id?orgId=1&access_token={token}"></iframe>
+                    <iframe src="https://grafana.example.com/d/dashboard-id?orgId=1&INGESTION_KEY={token}"></iframe>
```
Comment (therealpandey) [inline data/guides/how-to-authenticate-and-embedded-grafana-charts-into-iframe.mdx]: This iframe needed to be reverted.

File: data/opentelemetry/go.mdx [hunk 5]
```diff
@@ -322,7 +322,7 @@ Where:
     
 - **`OTEL_EXPORTER_OTLP_HEADERS`**:
     
-    This header contains the SigNoz access token for authentication. Replace `<SIGNOZ-INGESTION-TOKEN>` with your actual [ingestion token](https://signoz.io/docs/ingestion/signoz-cloud/keys/).
+    This header contains the SigNoz ingestion key for authentication. Replace `<SIGNOZ-INGESTION-TOKEN>` with your actual [ingestion token](https://signoz.io/docs/ingestion/signoz-cloud/keys/).
```
Comment (therealpandey) [inline data/opentelemetry/go.mdx]: SIGNOZ_INGESTION_KEY instead of SIGNOZ_INGESTION_TOKEN

**Review by therealpandey — CHANGES_REQUESTED** [review]

File: data/docs/alerts-management/terraform-provider-signoz.mdx (hunk 3 above)
Comment (ankit01-oss) [reply data/docs/alerts-management/terraform-provider-signoz.mdx]: @grandwizard28, should it be ACCESS_TOKEN = "api-key" or access_token = <api_key>. I mean to say no " " in there?

**Review by therealpandey — APPROVED** [review]

**Pull request merged**